m/milvus/milvus-v2.4.11-ubi9.3.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash -ex
+# ----------------------------------------------------------------------------
+#
+# Package       : milvus
+# Version       : 2.4.11
+# Source repo   : https://github.com/milvus-io/milvus
+# Tested on     : UBI 9.3 (docker)
```

Comment:
Lets remove the (docker) string for consistency

Reply:
We have tested it using docker, so its a good idea to inform.

Reply:
actually that's self implied when we say ubi 9.3

Reply:
I meant it wasnt podman.

Reply:
Okay, got it. Lets add  a comment in the script in that case, we don't want to complicate / make the text in these headers free form as currency / bulk search parses and uses the context from here.